### PR TITLE
Add options examples to find_in_batches method

### PR DESCRIPTION
**Pull request by davidauza-engineer**
### Summary

When reading documentation as a newbie, you always appreciate having many examples to understand a topic. That's why I'm adding some examples that were lacking in the find_in_batches method, so the reader has more material to understand properly the method and its available options.

### Other Information

Thank you, please let me know your comments.

Kind regards from Colombia,

[David Auza](https://github.com/davidauza-engineer)

### CODE REVIEW

File: guides/source/active_record_querying.md [hunk 1]
```diff
@@ -431,7 +431,41 @@ internally to iterate.
 
 ##### Options for `find_in_batches`
 
-The `find_in_batches` method accepts the same options as `find_each`.
+The `find_in_batches` method accepts the same options as `find_each`:
+
+**`:batch_size`**
+
+Just like for `find_each`, it allows establishing how many records will be retrieved in each batch. Retrieving batches of 2500 records can be specified like:
```
Comment (eileencodes) [inline guides/source/active_record_querying.md]:
```suggestion
Just like for `find_each`, `batch_size` establishes how many records will be retrieved in each group. For example, retrieving batches of 2500 records can be specified as:
```

File: guides/source/active_record_querying.md [hunk 2]
```diff
@@ -431,7 +431,41 @@ internally to iterate.
...
+
+```ruby
+Invoice.find_in_batches(batch_size: 2500) do |invoices|
+  export.add_invoices(invoices)
+end
+```
+
+**`:start`**
+
+This option allows specifying the starting ID from where records will be retrieved. Note that as mentioned before, by default records are fetched in ascending order of the primary key. For example, to retrieve invoices starting on ID: 5000 in batches of 2500 records, the following code can be used:
```
Comment (eileencodes) [inline guides/source/active_record_querying.md]:
```suggestion
The `start` option allows specifying the beginning ID from where records will be selected. As mentioned before, by default records are fetched in ascending order of the primary key. For example, to retrieve invoices starting on ID: 5000 in batches of 2500 records, the following code can be used:
```

File: guides/source/active_record_querying.md [hunk 3]
```diff
@@ -431,7 +431,41 @@ internally to iterate.
...
+
+```ruby
+Invoice.find_in_batches(batch_size: 2500, start: 5000) do |invoices|
+  export.add_invoices(invoices)
+end
+```
+
+**`:finish`**
+
+This option allows specifying the ending ID of the records to be retrieved. The code below shows the case of retrieving invoices in batches, up to the invoice with ID: 7000:
```
Comment (eileencodes) [inline guides/source/active_record_querying.md]:
```suggestion
The `finish` option allows specifying the ending ID of the records to be retrieved. The code below shows the case of retrieving invoices in batches, up to the invoice with ID: 7000:
```

File: guides/source/active_record_querying.md [hunk 4]
```diff
@@ -431,7 +431,41 @@ internally to iterate.
...
+
+```ruby
+Invoice.find_in_batches(finish: 7000) do |invoices|
+  export.add_invoices(invoices)
+end
+```
+
+**`:error_on_ignore`**
+
+Exactly like for `find_each`it overrides the application config to specify if an error should be raised when a specific order is present in the relation.
```
Comment (eileencodes) [inline guides/source/active_record_querying.md]: I don't think docs should assume you read `find_each`.

```suggestion
The `error_on_ignore` option overrides the application config to specify if an error should be raised when a specific order is present in the relation.
```

**Review by eileencodes — CHANGES_REQUESTED** [review]
This is looking good. I added some requested changes to tidy up the language a bit.

**Comment by davidauza-engineer** [comment]
Hi @eileencodes 

Thank you very much for your suggestions, I really appreciate them. In order to merge the changes should I squash the commits into a single one?

Thank you very much, I'll be waiting for your answer,

Best regards,

David Auza

**Comment by eileencodes** [comment]
> In order to merge the changes should I squash the commits into a single one

Yup, please squash and I'll merge. Thanks!

**Comment by davidauza-engineer** [comment]
Hi @eileencodes 

All the commits were squashed into a single commit ✅ 

Thank you very much for all your support

**Review by eileencodes — APPROVED** [review]

**Comment by eileencodes** [comment]
Thanks @davidauza-engineer!

**Pull request merged**